guacamole/src/licenses/LICENSE

```diff
@@ -605,6 +605,36 @@ licenses; we recommend you read them, as their terms may differ from the
 terms above.
 
 
+JSTZ (http://webjars.org/)
```

Comment:
The URL here should be the URL of the project.

Perhaps https://pellepim.bitbucket.io/jstz/?